README.md

```diff
@@ -51,6 +51,12 @@ emitter.on('foo', onFoo)   // listen
 emitter.off('foo', onFoo)  // unlisten
 ```
 
+### Typescript
+```ts
+import * as mitt from 'mitt';
+let emitter: mitt.Emitter = new mitt();
```

Comment:
is the `new` required for typescript here? `mitt()` is just a function (you can call it with `new`, but it just returns).